tests/test_config_consistency.py

```diff
@@ -327,7 +328,7 @@ class Test_Config_IntegrationEnabled_False:
     """ Verify behavior of integrations automatic spans """
 
     def setup_kafka_integration_enabled_false(self):
-        self.r = weblog.get("/kafka/produce")
+        self.r = weblog.get("/kafka/produce", params={"topic": "random"}, timeout=60)
```

Comment:
Why is there such a long timeout ?

Reply:
Based it off another test that had timeout as 60